vgen/src/main/java/io/gapi/vgen/csharp/CSharpContextCommon.java

```diff
@@ -83,7 +83,15 @@ public String addImport(String namespace) {
   // the functionality, since snippets can't call static functions.
   public String underscoresToCamelCase(
       String input, boolean capNextLetter, boolean preservePeriod) {
-    return s_underscoresToCamelCase(input, capNextLetter, preservePeriod);
+    return s_underscoresToCamelCase(input, capNextLetter, preservePeriod, false);
+  }
+
+  public static String s_underscoresToCamelCase(String input) {
```

Comment:
I can provide some context. This was done when the big Dicovery merge happened. The motivation is that it is not possible to call static methods from the snippet file, but static methods are easier to use when programming in plain Java (assuming, of course, that it can be made static). So, the s_method is the static versions, the instance methods simply delegate the call.